src/keymap.cpp

```diff
@@ -733,6 +733,14 @@ void KeyMap::handle_action(const std::string& action, const std::string& params)
 		} else {
 			unset_key(tokens[0], context);
 		}
+	} else if (action == "bind") {
```

Comment:
This has no user-observable effect yet until we register the keymap as a handler for "bind" in `Controller` and `PbController`:
https://github.com/newsboat/newsboat/blob/ca977d51a0ea9bc08d302d9ce58e424090823a87/src/controller.cpp#L168-L172

https://github.com/newsboat/newsboat/blob/ca977d51a0ea9bc08d302d9ce58e424090823a87/src/pbcontroller.cpp#L260-L261